app/controllers/user.js

```diff
@@ -39,6 +43,40 @@ exports.create = (req, res) => {
     });
 };
 
+exports.generate = (req, res) => {
+  const username = short.generate();
+  const cardKey = short.generate();
+  const password = crypto.randomBytes(10).toString('hex');
+  const unActivatedEmail = req.body.email;
+  const userObject = {
+    username,
+    password,
+    cardKey,
+    unActivatedEmail,
```

Comment:
We probably need a way to verify that this isn't a duplicate by having the separate "index" we talked about yesterday, otherwise this looks awesome! Great job!

Reply:
Yeah, haven't figured out the smoothest way to do this yet, open to suggestions. There is no global/singleton type scheme in VOTE, so I haven't figured out where to put it. 

The easiest way would be to just create a whole new collection, and that is maybe the best way too.